src/main/java/com/hll/hyperlightlogistics/controller/CustomerController.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of HyperLightLogistics-Java.
+ *
+ * HyperLightLogistics-Java is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HyperLightLogistics-Java is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HyperLightLogistics-Java.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Copyright (C) 2024 Vsevolod Batyrov
+ */
+
+package com.hll.hyperlightlogistics.controller;
+
+import com.hll.hyperlightlogistics.dto.AddressDTO;
+import com.hll.hyperlightlogistics.dto.CustomerRequestDTO;
+import com.hll.hyperlightlogistics.mapper.AddressMapper;
+import com.hll.hyperlightlogistics.model.CustomerAddress;
+import com.hll.hyperlightlogistics.repository.CustomerRepository;
+import com.hll.hyperlightlogistics.service.CustomerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/api/customers")
+public class CustomerController {
+
+    @Autowired
+    private CustomerService customerService;
+
+    @Autowired
+    private CustomerRepository customerRepository;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private AddressMapper addressMapper;
+
+    @PostMapping("/createCustomer")
+    public ResponseEntity<String> createCustomer(@RequestBody CustomerRequestDTO customerRequest) {
+
+        String response = customerService.createCustomer(customerRequest);
+
+        return ResponseEntity.ok(response);
+
+    }
+
+    @PostMapping("/{customerId}/addAddress")
+    public ResponseEntity<String> addAddressToCustomer(
+            @PathVariable Long customerId,
+            @RequestBody AddressDTO addressRequest) {
+
+        try {
+            customerService.addAddressToCustomer(customerId, addressRequest);
+            return ResponseEntity.ok("Address added successfully");
+        } catch (RuntimeException e) {
+            return ResponseEntity.badRequest().body(e.getMessage());
+
+        }
+    }
+
+    @GetMapping("/getAllAddresses/{customerId}")
```

Comment:
```suggestion
    @GetMapping("/{customerId}/addresses")
```
Just use the same mapping as for creation, but replace it with the HTTP GET method